bot/util/utilities.py

```diff
@@ -55,6 +55,37 @@ async def server_info(self, ctx):
         embed.add_field(name="Rollen :medal:", value=embed_strings[5], inline=True)
         await ctx.send(embed=embed)
 
+    @commands.command(name='about')
+    async def about(self, ctx):
+        """Command Handler for the `about` command.
+
+        Args:
+            ctx (Context): The context in which the command was called.
+
+        Returns:
+            Embed: An embedded message (Embed) containing some information about this bot and links regarding the
+                    GitHub repository and donations.
+        """
```

Comment:
Ich glaub man kann mit den dreifachen """ in python mehrzeilige strings machen

https://www.techbeamers.com/python-multiline-string/

Reply:
Yes, one could do that but the problem here is that we want to have a single line string so that the embed can adapt itself in regards to the device being used. The reason why I split the string in this particular way to begin with is just that the code is better formatted and therefore easier to read.